### ISSUE TITLE
Does [[clang::jit]] Attribute force a template to be JITted?

### ISSUE THREAD

**Issue by CaoZhongZ** [hfinkel/llvm-project-cxxjit#14]
I mean if I want a template to be hybrid, specialize it when I have decisive parameters and JIT it when I haven't.

do_something<i>(...); // This is JIT, OK.
do_something<3>(...); //  normal C++ behavior or JIT?

**Comment by hfinkel** [comment]
Yes, although I'm going to work soon on implementing a new interface for this which should allow for this level of control (see http://wg21.link/p1609, section "3. Updated Proposal" - feedback welcome). In the mean time, you can use a wrapper template for the cases that you'd like to JIT.
